test/cthulhu.jl

```diff
@@ -0,0 +1,3 @@
+using Cthulhu: CTHULHU_MODULE
+const Cthulhu = CTHULHU_MODULE[]
+const CC = Cthulhu.CC
```

Comment:
That's to ensure that we can do `using Compiler` and have tests switch to using the extension module and keep working.